openquake/commonlib/calculators/base.py

```diff
@@ -42,6 +42,8 @@
 class AssetSiteAssociationError(Exception):
     """Raised when there are no hazard sites close enough to any asset"""
 
+rlz_dt = numpy.dtype([('uid', (str, 200)), ('weight', float)])
```

Comment:
I am defining a new datatype, with static parameters, this is done usually at top level, no?